activitysim/examples/prototype_mtc_extended/test/regress/final_trips.csv

```diff
@@ -1,80 +1,85 @@
-1368292657,4171623,1810015,work,1,True,2,25,16,171036582,escort,30.234430836012045,8,WALK,9.029527074456235
-1368292658,4171623,1810015,work,2,True,2,21,25,171036582,work,,8,WALK,2.0014416307382055
-1368292661,4171623,1810015,work,1,False,2,7,21,171036582,work,34.72578612209499,23,WALK,2.5646380272501808
-1368292662,4171623,1810015,work,2,False,2,16,7,171036582,home,,23,WALK,9.584561374619746
-2464104641,7512514,2821179,eatout,1,True,1,13,8,308013080,eatout,,8,WALK,-1.171759971785514
-2464104645,7512514,2821179,eatout,1,False,1,8,13,308013080,home,,16,WALK,-1.238718768693438
-2464104857,7512514,2821179,shopping,1,True,1,6,8,308013107,shopping,,17,WALK,12.612248978887928
-2464104861,7512514,2821179,shopping,1,False,1,8,6,308013107,home,,19,WALK,12.322088998148224
-2464104881,7512514,2821179,social,1,True,1,9,8,308013110,social,,16,WALK,6.292424410910544
-2464104885,7512514,2821179,social,1,False,1,8,9,308013110,home,,16,WALK_LOC,6.322192231184283
-2464449633,7513565,2822230,work,1,True,2,9,8,308056204,univ,40.040196758213916,9,WALK,7.9948426686587775
-2464449634,7513565,2822230,work,2,True,2,25,9,308056204,work,,9,WALK,8.34752700809022
-2464449637,7513565,2822230,work,1,False,1,8,25,308056204,home,,18,WALK,9.31552710851258
+trip_id,person_id,household_id,primary_purpose,trip_num,outbound,trip_count,destination,origin,tour_id,escort_participants,school_escort_direction,purpose,destination_logsum,depart,trip_mode,mode_choice_logsum
+242323049,644476,386761,escort,1,True,1,21,16,30290381,,,escort,,6,WALK_LOC,4.042535128605572
+242323053,644476,386761,escort,1,False,1,16,21,30290381,,,home,,7,WALK_LOC,4.052167807394184
+242323057,644476,386761,escort,1,True,1,5,16,30290382,,,escort,,18,WALK_LOC,4.080648337226452
+242323061,644476,386761,escort,1,False,1,16,5,30290382,,,home,,19,WALK_LOC,3.988008105859652
+242323337,644476,386761,work,1,True,2,16,16,30290417,,,shopping,22.646840437961423,7,WALK,5.89391877387621
+242323338,644476,386761,work,2,True,2,4,16,30290417,,,work,,9,WALK_LOC,0.5894708670927961
+242323341,644476,386761,work,1,False,1,16,4,30290417,,,home,,14,WALK,0.5926711189721502
+242323617,644477,386761,escort,1,True,1,20,16,30290452,644478,outbound,escort,,9,WALK_LOC,2.2266987144854578
+242323621,644477,386761,escort,1,False,1,16,20,30290452,,,home,,10,WALK_LOC,2.0602667118131186
+242323625,644477,386761,escort,1,True,1,20,16,30290453,,,escort,,15,SHARED2FREE,0.02563756984130738
+242323629,644477,386761,escort,1,False,1,16,20,30290453,644478,inbound,home,,21,SHARED2FREE,-0.6067369442221502
+242323665,644477,386761,shopping,1,True,1,24,16,30290458,,,shopping,,12,WALK,2.6242779649960877
+242323669,644477,386761,shopping,1,False,1,16,24,30290458,,,home,,15,WALK,2.1220846081768716
+242323977,644478,386761,school,1,True,1,20,16,30290497,644478,outbound,school,,9,WALK_LOC,1.5478828044209911
+242323981,644478,386761,school,1,False,1,16,20,30290497,644478,inbound,home,,21,WALK_LRF,3.7659908084629183
```

Comment:
I notice here that there appears to be an inconsistency between trip mode choice of chauffeur and escortee.  Trip 242323629 is chauffeur escorting a child home from school, trip 242323981 is the child being escorted.  Purpose, destination, schedule all line up as intended.  But the chauffeur is driving (SHARED2FREE) while the child is using transit (WALK_LRF).  Reviewing the various code changes, it looks like this discrepancy is not a bug in code that has been written, but just something that has not been locked down; I don't recall if this is an intentional or unintentional omission.

Reply:
Post-processing functionality has been added to force escortee tour and trip modes to match the chauffeur modes.  For tour modes, there is ambiguity as the tour mode for the outbound leg can be different for the inbound leg since the pickup/dropoff can happen on different chauffeur tours. (The current implementation just uses the last available chauffeur tour mode.  Maybe a mode heirarchy should be implemented? But this would have to change between modeling regions...) I am not sure it really matters considering the trip modes are made to match downstream.